idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -298,6 +298,21 @@ service WorkflowService {
       4: shared.ServiceBusyError serviceBusyError,
     )
 
+  /**
+  * SignalWithStartWorkflowExecution is used to ensure sending a signal event to a workflow execution.
```

Comment:
SignalWithStartWorkflowExecution is used to ensure sending a signal event to a workflow execution. ->
SignalWithStartWorkflowExecution is used to send a signal event to an executed workflow.